js/notification.js

```diff
@@ -1,34 +1,21 @@
 const notifier = require('node-notifier');
 const path = require('path');
 
-/**
- * Notify user it's time to leave.
- */
-function notifyUser() {
-    notifier.notify(
-        {
-            title: 'Time to leave',
-            message: 'Hey there! I think it\'s time to leave.',
-            icon: path.join(__dirname, 'assets/timer.png'), // Absolute path (doesn't work on balloons)
-            sound: true, // Only Notification Center or Windows Toasters
-            wait: true // Wait with callback, until user action is taken against notification
-        },
-      );
-}
+const title = 'Time to Leave';
+const defaultMessage = 'Hey there! I think it\'s time to leave.';
 
 function notify(msg) {
     notifier.notify(
         {
-            title: 'Time to leave',
-            message: msg,
+            title: title,
+            message: msg || defaultMessage,
             icon: path.join(__dirname, 'assets/timer.png'), // Absolute path (doesn't work on balloons)
             sound: true, // Only Notification Center or Windows Toasters
-            wait: false // Wait with callback, until user action is taken against notification
+            wait: !msg // Wait with callback, until user action is taken against notification
```

Comment:
If #73 proves to be true. We can use the following changes here.
```suggestion
            wait: true
```